app/src/main/java/org/covidwatch/android/ui/menu/MenuFragment.kt

```diff
@@ -44,4 +57,34 @@ class MenuFragment : Fragment(R.layout.fragment_menu) {
         val browserIntent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
         startActivity(browserIntent)
     }
-}
\ No newline at end of file
+
```

Comment:
As I mentioned above,  move all the logic that you added here to `ExposureInformationRepository `

Reply:
Done